pkg/controller/amazoncloudintegration/amazoncloudintegration_controller.go

```diff
@@ -194,18 +220,18 @@ func (r *ReconcileAmazonCloudIntegration) Reconcile(ctx context.Context, request
 	}
 	component, err := render.AmazonCloudIntegration(amazonCloudIntegrationCfg)
 	if err != nil {
-		r.SetDegraded("Error rendering AmazonCloudIntegration", err, reqLogger)
+		r.SetDegraded(operatorv1.ResourceRenderingError, "Error rendering AmazonCloudIntegration", err, reqLogger)
```

Comment:
This err is hard-coded as nil. I think you could change `render.AmazonCloudIntegration(...)` such that only one param is returned. And then remove this if statement.

Reply:
Done